src/domain/group/component/InputGroup.js

```diff
@@ -64,19 +65,26 @@ class InputGroup extends React.Component {
               return (
                 <form className="form-inline" key={key}>
                   {matches ? [
-                    <label style={{ marginBottom: '0', marginRight: '.5rem' }} key="label">Group ID</label>,
+                    <label style={{ marginBottom: '0', marginRight: '.5rem' }} key={uuid.v4()}>Group ID</label>,
```

Comment:
@reyhansofian why we need to use `uuid` yea? `key` is different from `id` btw. Do you have any reference for this? If not, I prefer we drop the `uuid` part and its dependency as well. The more dependencies we use, it will be harder to maintain.